### PR TITLE
MM-21374 - Fixed extracting icon for prepackaged and local plugins

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Fixed path used to extract prepackaged plugin icon and also included icon data for local installed plugins. Icons are now displayed properly in Plugins Marketplace for local and prepackaged plugins.

#### Ticket Link
[MM-21374](https://mattermost.atlassian.net/browse/MM-21374)
[MM-21516](https://mattermost.atlassian.net/browse/MM-21516)

### CODE REVIEW

**Comment by marianunez** [comment]
@hanzei heads up that I updated here the bundle for the test plugins to add icon data. We need to coordinate the merge with https://github.com/mattermost/mattermost-server/pull/13595 so neither replaces the other without updating the new fields in the manifests of those bundles.

**Comment by marianunez** [comment]
@hanzei I've updated the plugin bundles and signatures here to include also your changes for `homepageURL`. It's ready for review 👀

File: report.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites></testsuites>
```
Comment (hanzei) [inline report.xml]: Why did you add this file?

Comment (marianunez) [reply report.xml]: Ugh shouldn't have gone in. Not sure if this is something we want to add to the `.gitignore`

Comment (hanzei) [reply report.xml]: How did the file got generated?
